packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx

```diff
@@ -136,19 +176,30 @@ const VisitNotesForm: React.FC<VisitNotesFormProps> = ({ patientUuid, closeWorks
     };
 
     const ac = new AbortController();
-    saveVisitNote(ac, visitNotePayload).then((response) => {
-      if (response.status === 201) {
-        closeWorkspace();
+    saveVisitNote(ac, visitNotePayload)
+      .then((response) => {
+        if (response.status === 201) {
+          closeWorkspace();
+
+          showToast({
+            kind: 'success',
+            description: t('visitNoteSaved', 'Visit note saved successfully'),
+          });
+        }
+      })
+      .catch((err) => {
         showToast({
```

Comment:
Toast or notification?

Reply:
Inline notification. I'll restore that block of code once we put out a new version of `esm-framework` with the notification implementation.